cmd/devstream/install.go

```diff
@@ -44,11 +44,12 @@ func installCMDFunc(cmd *cobra.Command, args []string) {
 
 	errs := p.Execute()
 	if len(errs) == 0 {
-		log.Println("=== all plugins' installation are succeeded ===")
+		log.Println("=== all plugins' Install/Uninstall/Reinstall process are succeeded ===")
```

Comment:
This error message is a little bit ambiguous thus not quite helpful from the user's standpoint.

It's possible to put the actual action (install, uninstall) into the errors. Please create a ticket in the backlog (for v0.0.1 we will not implement).

Reply:
@daniel-hutao

Reply:
Okay, I will keep track of this and continue to improve later.

Reply:
#49